.prettierignore

```diff
@@ -8,8 +8,11 @@
 !.vscode/**/
 !Apps/**/
 !Documentation/**/
-!Source/**/
+!packages/engine/Source/**/
```

Comment:
I don't believe prettier is being run on these files. I think you will need to adjust the globs to unignore directories to all depths as the comment says above.